owncloudApp/src/main/java/com/owncloud/android/ui/activity/FolderPickerActivity.kt

```diff
@@ -155,6 +159,18 @@ open class FolderPickerActivity : FileActivity(),
 
     override fun onCreateOptionsMenu(menu: Menu?): Boolean {
         menuInflater.inflate(R.menu.main_menu, menu)
+        menu?.let {
+            it.findItem(R.id.action_search)?.let { menuItem ->
+                (menuItem.actionView as? SearchView)?.apply {
```

Comment:
Why using `as?`? There's no need to in this case 😀

Reply:
I changed that and now it's `as` instead `as?` ✅